### ISSUE TITLE
Create common pronoun files

### ISSUE THREAD

**Issue by TrangPham** [emdaniels/character-swap#2]
I think you could extract very common pronoun example files for people to start with.

i.e. a lot of the entries in: https://github.com/emdaniels/character-swap/blob/master/Frankenstein/he_frankenstein_pronouns.csv
could be extracted out to he_to_she_pronouns_example.csv

**Comment by emdaniels** [comment]
You are right- I've been thinking to make it into a sort of static dictionary object once to eliminate the replication. Thanks for mentioning it and reminding me!

**Comment by emdaniels** [comment]
Hey @TrangPham, just updated the project and created common pronoun files, as well as some other functionality that will make it easier to run. Let me know what you think!
